mediorum/server/serve_blob.go

```diff
@@ -59,7 +60,7 @@ func (ss *MediorumServer) getBlob(c echo.Context) error {
 			return err
 		}
 		defer blob.Close()
-		return c.Stream(200, blob.ContentType(), blob)
+		http.ServeContent(c.Response(), c.Request(), key, blob.ModTime(), blob)
```

Comment:
need to add `return nil` here so doesn't fall thru to redirect.

Also I see lower down that we allow client to specify a filename... do we want to use that here if specified or `key` otherwise?

Reply:
added `return nil` and filename option for track download. tested track download as well. that's a good point because I do see in client code where it calls /ipfs with a `filename` queryparam to download it so now that flow is supported here